browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -150,6 +150,14 @@ class NavigatorPanel extends SidebarBase {
 			'keydown',
 			function (e: KeyboardEvent) {
 				if (e.code === 'Escape') {
+					const contextMenu = document.querySelector(
```

Comment:
this might be useful in many places, can we move to UIManager as a method?
```
const contextMenu = app.map.uiManager.isAnyContextMenuOpened();
if (contextMenu) return;
```